### PR TITLE
Add observability, reporting, query workbench and gantt chart to manifests

### PR DESCRIPTION
**Pull request by mengweieric**
### Description
Add observability, reporting and gantt chart to manifests

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: manifests/2.5.0/opensearch-2.5.0.yml [hunk 1]
```diff
@@ -108,7 +108,6 @@ components:
     platforms:
       - linux
       - windows
-    working_directory: opensearch-observability
```
Comment (zelinh) [inline manifests/2.5.0/opensearch-2.5.0.yml:111]: Have we removed all front end folders in this repo and make it exclusive for backend? If not, removing the `working_directory` may cause build failure.

File: manifests/2.5.0/opensearch-dashboards-2.5.0.yml [hunk 2]
```diff
@@ -22,6 +22,12 @@ components:
     working_directory: dashboards-notifications
     ref: 2.x
   - name: observabilityDashboards
-    repository: https://github.com/opensearch-project/observability.git
-    working_directory: dashboards-observability
+    repository: https://github.com/opensearch-project/dashboards-observability.git
+    ref: 2.x
+  - name: reportsDashboards
+    repository: https://github.com/opensearch-project/dashboards-reporting.git
```
Comment (zelinh) [inline manifests/2.5.0/opensearch-dashboards-2.5.0.yml:28]: This repo is still not public yet therefore cause the manifest CI checks failed.

Comment (rupal-bq) [reply manifests/2.5.0/opensearch-dashboards-2.5.0.yml:28]: This repo is public now

File: manifests/2.5.0/opensearch-2.5.0.yml (hunk 1 above)
Comment (mengweieric) [reply manifests/2.5.0/opensearch-2.5.0.yml:111]: there's a PR ready to merge which removes all front end stuff, will get it merged soon

File: manifests/2.5.0/opensearch-dashboards-2.5.0.yml (hunk 2 above)
Comment (ps48) [reply manifests/2.5.0/opensearch-dashboards-2.5.0.yml:28]: @zelinh Can you please re-run the failing tests?

Comment (zelinh) [reply manifests/2.5.0/opensearch-dashboards-2.5.0.yml:28]: @mengweieric Could you rebase to resolve the conflicts?

**Review by peterzhuamazon — APPROVED** [review]

**Comment by zelinh** [comment]
CI is failing as OpenSearch has bumped to 2.6.0 for 2.x branch. Will need to rebase after this PR merged. https://github.com/opensearch-project/opensearch-build/pull/3059

**Comment by anirudha** [comment]
@peterzhuamazon @zelinh can we review and merge this soon ?

**Comment by zelinh** [comment]
> @peterzhuamazon @zelinh can we review and merge this soon ?

@anirudha Yes, once the CI checks completed, I could help merge this.

**Pull request merged**